packages/stream-model-handler/yarn.lock

```diff
@@ -0,0 +1,845 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
please remove this file.  Fine if you use yarn locally, but the "official" way to build ceramic is with npm so we don't want to check in files for other build systems into the repo